openvisualizer/main.py

```diff
@@ -493,6 +516,13 @@ def _add_parser_args(parser):
         help='port number'
     )
 
+    parser.add_argument(
```

Comment:
Don't use "list_from_string", something like this should work I think:    

parser.add_argument(
        '--port-mask',
        dest='port_mask',
        type=str,
        action='store',
        nargs='+',
        help='List of baudrates to probe for, e.g \'115200 500000\''
    )